SA3/qa_web/templates/qa_web/answerspage.html

```diff
@@ -95,5 +120,12 @@ <h3>Be the first to answer this question!</h3>
   {% else %}
    <p>To answer this question, <a href="/login">login</a> or <a href="/signup">sign up</a>!</p>
   {% endif %}
+</div>
+</div>
+<div align="center" style="vertical-align:bottom">
+      <div class="bg-circle-1 bg-circle"></div>
+      <div class="bg-circle-2 bg-circle"></div>
+      <div class="bg-circle-3 bg-circle"></div>
+      <div class="bg-circle-4 bg-circle"></div>
   </body>
 </html>
```

Comment:
It is a good practice that we put the newline as a last character. (seems that many people met this red circle arrow symbol. https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file)